### DISCUSSION TITLE
The value of c is empty  in Extend with JavaScript - Routing

### DISCUSSION THREAD

**Discussion by getch04** [pocketbase/pocketbase#3211] [Q&A]
This was my code and I am passing a sample array body and both the response and console log  shows that the value of c is an empty object ({})

```
routerAdd("POST", "/api/movies", (c) => {
   console.log(JSON.stringify(c));
  return c.json(200, { data: c });
});
```

**Comment by ganigeorgiev** [comment]
`c` is `echo.Context` and it is an interface, it is not empty. The string representation could be empty because it doesn't have public fields to print. For more details about the argument, you can check https://pocketbase.io/jsvm/interfaces/echo.Context.html.
